include/deal.II/fe/mapping_q_generic.h

```diff
@@ -19,930 +19,7 @@
 
 #include <deal.II/base/config.h>
 
-#include <deal.II/base/derivative_form.h>
-#include <deal.II/base/polynomial.h>
-#include <deal.II/base/quadrature_lib.h>
-#include <deal.II/base/table.h>
-#include <deal.II/base/vectorization.h>
+#include <deal.II/fe/mapping_q.h>
```

Comment:
Do we need the following?

```suggestion
#include <deal.II/fe/mapping_q.h>

template<int dim, int spacedim>
using MappingQGeneric = MappingQ<dim, spacedim>
```